### PR TITLE
React Native installation guide for windows

### PR DESCRIPTION
**Pull request by kuldeep3**
**Changes included**

1. Added installation guide for setting up a mobile development environment on the windows machine.

### CODE REVIEW

File: react-native/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# React Native
+
+This document covers installation guidelines that helps developer to setup react native development environment locally.
+
+### Windows
+1. [React Native Guide](/react-native/WINDOWS.md)
```
Comment (Satendra-SR) [inline react-native/README.md:6]:
```suggestion
1. [React Native Installation and Setup Guide](/react-native/WINDOWS.md)
```

File: react-native/WINDOWS.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+1. Android Studio.
+
+### Installing Node
+
+1. [Download Node](https://nodejs.org/en/download/) (choose Node version LTS/Current).
+1. Run the installer.
+1. Verify the installation by running
+```zsh
+node --v
+npm --v
```
Comment (Satendra-SR) [inline react-native/WINDOWS.md:15]: Shouldn't it be like this?
```suggestion
node -v
npm -v
```

Comment (Satendra-SR) [reply react-native/WINDOWS.md:15]: Or `node --version`?

File: react-native/WINDOWS.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Installation
+
+### Requirements
+
+1. Node
+1. Android Studio.
+
+### Installing Node
+
+1. [Download Node](https://nodejs.org/en/download/) (choose Node version LTS/Current).
```
Comment (Satendra-SR) [inline react-native/WINDOWS.md:10]: Does `npm` come with `node`?

File: react-native/WINDOWS.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+1. After proper installation, add virtual device by navigating to the `AVD manager` located at the top-right corner of the toolbar with an android icon.
+1. Follow the steps as suggested by the `AVD manager` to set up the virtual device on the application.
+1. After successfully adding a virtual device to the application, navigate to the `SDK manager` located besides `AVD manager`.
+1. Choose `Android 10(R and Q)` or `greater than Android 10` both from `SDK platforms tab`, and move to the `SDK tools tab`.
+1. Select `SDK Build-tools`, `NDK(side-by-side)`, `Android SDK Command-line Tools (Latest)`, `Android Emulator` followed by `Android SDK Platform-tools`, `Google Play APK Expansion Library`, `Google Play Instant Development SDK`, `Google Play Licensing Library`, `Google Play Services` and `Intel x86 Emulator Accelerator(HAXM installer)` or `Android Emulator Hypervisor Driver for AMD Processors` (optional depends upon the machine).
+```zsh
+NOTE: Maximum tools will be installed by default.
+```
+7. After selecting all the appropriate options, install and hit `apply` followed by the `ok` button.
```
Comment (Satendra-SR) [inline react-native/WINDOWS.md:29]:
```suggestion
1. After selecting all the appropriate options, install and hit `apply` followed by the `ok` button.
```

File: react-native/WINDOWS.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+# Installation
+
+### Requirements
+
+1. Node
```
Comment (Satendra-SR) [inline react-native/WINDOWS.md:5]: What about `npm` and `npx`?

**Review by Satendra-SR — COMMENTED** [review]
@kuldeep3 a few minor questions and suggestions. Have a look.

File: react-native/WINDOWS.md (hunk 3 above)
Comment (kuldeep3) [reply react-native/WINDOWS.md:10]: @Satendra-SR Yes, it comes with the node

File: react-native/WINDOWS.md (hunk 5 above)
Comment (kuldeep3) [reply react-native/WINDOWS.md:5]: @Satendra-SR npm comes with the node and npx can be installed via npm, which I have mentioned in the `working with the project` section.

**Review by Satendra-SR — APPROVED** [review]
Looks good 👍

**Pull request merged**